src/stk/molecular/topology_graphs/host_guest/complex.py

```diff
@@ -6,6 +6,71 @@
 
 from .vertices import HostVertex, GuestVertex
 from ..topology_graph import TopologyGraph, NullOptimizer
+import warnings
+
+
+class Guest:
+    """
+    Holds the data defining the placement of a guest molecule.
+
+    """
+
+    def __init__(
+        self,
+        building_block,
+        start_vector=(1., 0., 0.),
+        end_vector=(1., 0., 0.),
+        displacement=(1., 0., 0.),
+    ):
+        """
+        Initialize a :class:`.Guest` instance.
+
+
+        Parameters
+        ----------
+        building_block : :class:`.BuildingBlock`
+            The guest molecule.
+
+        start_vector : :class:`tuple` of :class:`float`, optional
+            A direction vector which gets aligned with `end_vector`.
+
+        end_vector : :class:`tuple` of :class:`float`, optional
+            A direction vector which determines the rotation applied to
+            the `building_block`. A rotation such that
+            `start_vector` is transformed into `end_vector`
+            is applied.
+
+        displacement : :class:`tuple` of :class:`float`, optional
+            The translational offset of the guest.
+
+
+        """
+
+        self._building_block = building_block
+        self._start_vector = start_vector
+        self._end_vector = end_vector
+        self._displacement = displacement
+
+    def get_building_block(self):
```

Comment:
These methods needs docstrings :)

Reply:
I want that emoji that is a smile with a tear in its eyes.